bench/macro/lsm-tree-bench-wp8.hs

```diff
@@ -275,6 +280,13 @@ doDryRun' gopts opts = do
         duplicates <- readIORef duplicateRef
         printf "True duplicates: %d\n" duplicates
 
+    -- See batchOverlaps for explanation of this check.
+    when opts.check $
+        let anyOverlap = (not . null)
+                           (batchOverlaps gopts.initialSize opts.batchSize
+                                          opts.batchCount opts.seed)
+         in putStrLn $ "Any adjacent batches with overlap: " ++ show anyOverlap
```

Comment:
Would it be informative if we count the number of overlapping keys/batches instead?

Reply:
Slightly, but then it's slower to calculate. And the important thing is that there's at least one, since that gives us coverage of the pipelined method of calculating the correct lookup results.